### PR TITLE
Desktop: Added ability to see the complete address when initiating a transfer.

### LINKED ISSUE #474: Add ability so see the complete address when initiating a transfer
### Description
Not sure if needed, I personally would appreciate a feature like this: When sending a TX the user is not able to see the whole recipient address. Hovering the address field could show the whole address in a hovering textfield which is attached to the cursor. See screenshot for very poor example.

### Environment Specs
Trinity Desktop

### Motivation
its more userfriendly because the user can directly see if he entered everything correct without the need to try to send the tx, then notice he made a mistake in the confirmation screen just to go back, try to fiddle around in the small input field and then try again.

### Am I planning to do it myself with a PR?
no

45118926-30480780-b15a-11e8-9ada-d655e6db0920

### PR DESCRIPTION
**Pull request by mfecenko**
# Description

Added ability to see the complete address when initiating a transfer. Address is displayed in tooltip, which is attached to cursor and displayed only when input si not focused and address is longer than input width.

Closes #474

screen1

## Type of change

- Enhancement (a non-breaking change which adds functionality)

# How Has This Been Tested?

- Manually simulated typing of address and checking if tooltip displays when it should and if it's displayed correctly.

# Checklist:

- [X] My code follows the style guidelines for this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by rajivshah3 — APPROVED** [review]
Works great! Thanks @mfecenko

File: src/desktop/src/ui/components/input/input.scss [hunk 1]
```diff
@@ -540,3 +540,20 @@
...
+        border-bottom-color: var(--box-bg) !important;
+    }
+}
+
+.addressHidden {
+    visibility: hidden;
+    font-size: 14px;
+    padding-left: 10px;
+    padding-right: 40px;
+}
```
Comment (rihardsgravis) [inline src/desktop/src/ui/components/input/input.scss:564]: I run this by Andrew (Trinity designer) and he suggested using other colors for better contrast, split the address in multiple lines for better readability, use `Source Code Pro` font and some minor layout updates:
```
.tooltip {
    background: var(--bar-bg) !important;
    color: var(--bar) !important;
    width: 324px !important;
    font-family: 'SourceCodePro';
    letter-spacing: 2px;
    word-wrap: break-word;
    line-height: 21px;
    padding: 12px 14px 12px 16px !important;

    &:after {
        border-bottom-color: var(--bar-bg) !important;
    }
}
```

**Review by rihardsgravis — COMMENTED** [review]
Nicely done! 👌

File: src/desktop/src/ui/components/input/input.scss (hunk 1 above)
Comment (mfecenko) [reply src/desktop/src/ui/components/input/input.scss:564]: updated the tooltip style

**Pull request merged**